autora/cycle/__init__.py

```diff
@@ -1,3 +1,9 @@
 from .simple import SimpleCycle as Cycle
 
-from .plot_utils import plot_results_panel_2d, plot_results_panel_3d  # isort:skip
+from .plot_utils import (  # isort:skip
```

Comment:
why do we have the isort:skip here? it shouldn't make a difference – the imports look to be imported in the right order.

Reply:
Ah. It will try to sort with .plot_utils before .simple imports (because alphabetical). This causes an import error because the plot_utils imports Cycle. Perhaps plot_utils shouldn't import Cycle but instead import SimpleCycle?

Reply:
Ah, I see. A more fundamental question is whether we should import the members of .plot_utils at all, or whether you should just do this in your analysis script:
```python
from autora.cycle import Cycle  # for the default cycler, or
from autora.cycle.simple import SimpleCycle  # if you're wanting the SimpleCycle explicitly
from autora.cycle.plot import plot_cycle_score
```

Reply:
Yeah I suppose we could do it that way. I was trying to keep it similar to how we import plot utils for autora.theorist.darts but I'm not sure if there are examples of those plotters being used anywhere. The `__init__` for darts does have a `from .visualize import darts_model_plot`. I think that's for the model graph though. Thoughts?

Reply:
I changed the import statement in the plot_utils to import Cycle from simple directly. This avoids the import error and needing the isort:skip.